internal/metrics/metricsManager.go

```diff
@@ -115,7 +115,7 @@ func NewMetricsManager(registry prometheus.Registerer) *MetricsManager {
 			Subsystem: subsystem,
 			Name:      counterClusterHostInstallationCount,
 			Help:      counterDescriptionClusterHostInstallationCount,
-		}, []string{clusterIdLabel, emailDomainLabel}),
+		}, []string{openshiftVersionLabel, clusterIdLabel, emailDomainLabel}),
```

Comment:
you should provide the openshift version in [line#209](https://github.com/openshift/assisted-service/pull/870/files#diff-6d51a205c8e038996615c6ed4baa7cb8822fe75ef259ad398f66be80097f8600R209)